src/pages/cli/graphql/custom-business-logic.mdx

```diff
@@ -480,7 +480,8 @@ The `@http` transformer will create one HTTP datasource for each identified base
 
 ## VTL resolver
 
-You can use AWS Cloud Development Kit (CDK) to define custom VTL resolvers for your GraphQL API.
+You can use AWS Cloud Development Kit (CDK) to define custom VTL resolvers for your GraphQL API. `@auth` directives are not supported for custom queries or mutations that are connected to a VTL resolver. This is because you replacing Amplify's auto-generated capabilities for that particular query or mutation with a custom-defined cloud resources. 
```

Comment:
```suggestion
You can use AWS Cloud Development Kit (CDK) to define custom VTL resolvers for your GraphQL API. `@auth` directives are not supported for custom queries or mutations that are connected to a VTL resolver. This is because you are replacing Amplify's auto-generated capabilities for that particular query or mutation with a custom-defined cloud resources. 
```